darshan-util/pydarshan/pyproject.toml

```diff
@@ -1,11 +1,13 @@
 [build-system]
 requires = [
     "wheel",
-    "setuptools<64.0.0",
+    "setuptools>=65.0.0",
 ]
 
 [project]
+name = "darshan"
 requires-python = ">=3.7"
+version = "3.4.4.0"
```

Comment:
many projects will tend to use version numbers that have development indicators like `3.4.4.0.dev0` to avoid confusion with i.e., the final release or previous release with that same number, but I thought it best to just follow our current convention and paste this in from `setup.py` for now.

There may not be a compelling reason to change it anyway, unless for example we became so popular that we needed nightly or weekly wheels, where the dev tag would have a more compelling reason to exist to distinguish from stable releases.